test-resources/kind-config.yaml

```diff
@@ -1,6 +1,7 @@
-# See CONTRIBUTING.md -> "Running End-to-End Tests on kind" for instructions to run e2e tests on a kind cluster.
+# See CONTRIBUTING.md -> "Semi-Manual Test Scenarios" and "Running End-to-End Tests on kind" for instructions to run
```

Comment:
_question:_ Should we remove this file and only keep the kind config with the local registry? If we do not want to keep support for `kind docker load ...`, having a local kind cluster without a registry is not super useful, is it?